.yamato/build.yml

```diff
@@ -17,9 +20,12 @@ test_on_osx:
     type: Unity::VM::osx
     image: build-system/unity-extra-macos-10.13-katana:v0.3.6-621005
     flavor: b1.xlarge
+  variables:
+    ANDROID_SDK_ROOT: /Users/bokken/android-sdk_auto
   commands:
-    - perl build-osx.pl
-    - make -f test/Makefile
+    - perl build.pl
+    - ./bee build:osx:test
+    - ./build/osx/test
 
 test_trigger:
   name: Tests Trigger
```

Comment:
By the way, I noticed that when validating PR, we don't run Build JNIBridge config, thus if you want check builds.zip, you have to manually trigger the build. Is this something we should fix?

Reply:
I don't know. Building all 4 archs takes quite a while. The OSX tests are there so you have a red PR if they fail. As for android I think you are more likely to build and test them locally, using Yamato for final build.